### ISSUE TITLE
Crash when attempting to launch the Editor Demo

### ISSUE THREAD

**Issue by shahla00** [wordpress-mobile/AztecEditor-iOS#868]
### Expected behavior
The app displays the editor.

### Actual behavior
The app crashes.

### Steps to reproduce the behavior
1- Run the Aztec example app on iPhone 8 plus simulator running iOS 11.2.
2- Tap on "Editor Demo"
3- Notice how the app crashes at line 20 in PreFormatter.swift. The following is printed in the console:" Message: Unexpected end tag : p"

##### Tested on [device], iOS [version], Aztec [version]
iPhone 8 plus simulator running iOS 11.2

**Comment by brbrr** [comment]
Here some chunk of backtrace:
```
(lldb) bt
* thread #1, queue = 'com.apple.main-thread', stop reason = EXC_BAD_ACCESS (code=2, address=0x7ffee036b9a8)
    frame #0: 0x0000000110915603 CoreFoundation`CFDictionaryCreateMutableCopy + 99
    frame #1: 0x0000000118bee976 CoreText`TCFMutableDictionary::TCFMutableDictionary(__CFDictionary const*, long) + 38
    frame #2: 0x0000000118c0ab75 CoreText`TDescriptor::TDescriptor(__CFDictionary const*, unsigned int) + 79
    frame #3: 0x0000000118bedd50 CoreText`CTFontDescriptorCreateWithAttributesAndOptions + 86
    frame #4: 0x0000000117746a75 UIFoundation`-[UIFontDescriptor _swapWithFontAttributes:options:] + 71
    frame #5: 0x000000011774715b UIFoundation`+[UIFontDescriptor fontDescriptorWithName:size:] + 155
    frame #6: 0x0000000113b7a534 Aztec`@nonobjc UIFontDescriptor.__allocating_init(name:size:) at PreFormatter.swift:0
  * frame #7: 0x0000000113b7a4c3 Aztec`default argument 0 of PreFormatter.init(monospaceFont:placeholderAttributes:) at PreFormatter.swift:20
    frame #8: 0x0000000113b226ad Aztec`AttributedStringSerializer.init(defaultAttributes=0 key/value pairs) at AttributedStringSerializer.swift:165
    frame #9: 0x0000000113b23cce Aztec`AttributedStringSerializer.init() at AttributedStringSerializer.swift:17
    frame #10: 0x0000000113b23c44 Aztec`AttributedStringSerializer.__allocating_init() at AttributedStringSerializer.swift:0
    frame #11: 0x0000000113b37bcb Aztec`FigureElementConverter.init() at FigureElementConverter.swift:8
    frame #12: 0x0000000113b37b5a Aztec`FigureElementConverter.__allocating_init() at FigureElementConverter.swift:0
    frame #13: 0x0000000113b2297c Aztec`AttributedStringSerializer.init(defaultAttributes=0 key/value pairs) at AttributedStringSerializer.swift:178
    frame #14: 0x0000000113b23cce Aztec`AttributedStringSerializer.init() at AttributedStringSerializer.swift:17
    frame #15: 0x0000000113b23c44 Aztec`AttributedStringSerializer.__allocating_init() at AttributedStringSerializer.swift:0
    frame #16: 0x0000000113b37bcb Aztec`FigureElementConverter.init() at FigureElementConverter.swift:8
    frame #17: 0x0000000113b37b5a Aztec`FigureElementConverter.__allocating_init() at FigureElementConverter.swift:0
    frame #18: 0x0000000113b2297c Aztec`AttributedStringSerializer.init(defaultAttributes=0 key/value pairs) at AttributedStringSerializer.swift:178
```

**Comment by jleandroperez** [comment]
Fixed via #874 (pending review). Thank you @shahla00 and @brbrr!!
